Core/DataTypes/StructuredList.cs

```diff
@@ -40,6 +40,14 @@ public StructuredList(Type type)
         public abstract void Write(JsonTextWriter writerNotUsed);
 
         public abstract StructuredList Read(JsonTextReader reader);
+        
+
+        protected static readonly JsonLoadSettings LoadSettings = new()
```

Comment:
I didn't stumble of the definition of JsonLoadSettings. Does this come from NewtonSoft?

Reply:
yes it does, maybe it's new 🤷‍♂️